README.md

```diff
@@ -1,14 +1,15 @@
-
 # Fishjam iOS Client
+
 [Fishjam](https://github.com/fishjam-dev/fishjam) Client library for iOS apps written in Swift.
 
 ## Components
 
-The repository consists of 3 separapable components:
+The repository consists of 3 separate components:
 
-- `JellyfishClientSdk` - Fishjam client fully compatible with `Fishjam`, responsible for exchaning media events and receiving media streams which then are presented to the user
+- `JellyfishClientSdk` - Fishjam client fully compatible with `Fishjam`, responsible for exchanging media events and
```

Comment:
Imo before release all files and namespaces should be renamed to fishjam.

Reply:
Threre is another PR that does that: #20